### DISCUSSION TITLE
deployed to Heroku but can't login

### DISCUSSION THREAD

**Discussion by ethanturtle2002** [doccano/doccano#1357] [Q&A]
There's a chance I didn't do this right, but I tried the 1-click deployment to Heroku (so all I have done is basically pressed a button) but am not able to login at the doccano login screen once I open the app. I used "admin" and "password"

It returns "Incorrect username or password, or something went wrong."

ADMIN_USERNAME: "admin"
ADMIN_PASSWORD: "password"

What additional steps should I be taking?

**Comment by Hironsan** [comment]
I can't reproduce this problem. I can log in with the specified username and password.
Please check them set correctly.
